### PR TITLE
Updated variables in model

### PR DESCRIPTION
**Pull request by YangLeoZhao**
@pjaey Review plox :D

### CODE REVIEW

File: le_pr_delivery/delivery/models.py [hunk 1]
```diff
@@ -16,77 +16,77 @@ def is_authenticated(self):
...
+	category = models.TextField()
+	last_Used = models.DateTimeField()
+	total_Rating = models.IntegerField()
+	total_Count = models.IntegerField()
 
 	def __names__(self):
-		return self.Name
+		return self.name
 
 	def __Link__(self):
```
Comment (shahpranjal) [inline le_pr_delivery/delivery/models.py:44]: missed one  **Link** - > **link**

File: le_pr_delivery/delivery/views.py [hunk 2]
```diff
@@ -22,8 +23,18 @@ def mainPage(request):
...
 
 
-def bootstrap_css(request):
-	return render_to_response('le_pr_delivery/bootstrap.css')
+def restaurant(request):
+	return_list = []
+	all_restaurants = Restaurants.objects.all()
+	for restaurant in all_restaurants:
+		return_list.append(restaurant.__names__())
+		return_list.append(',')
```
Comment (shahpranjal) [inline le_pr_delivery/delivery/views.py:31]: should we keep this in /lib?

Comment (YangLeoZhao) [inline le_pr_delivery/delivery/views.py:31]: Yes i will refactor this into lib

File: le_pr_delivery/delivery/models.py (hunk 1 above)
Comment (YangLeoZhao) [inline le_pr_delivery/delivery/models.py:44]: Good catch

**Comment by shahpranjal** [comment]
approved after changes

**Pull request merged**